nodejs/week1/src/backend/routes/meals.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const filepath = require('path').resolve(__dirname, '..');
+
+
+function getReview(id) {
+    let revData = fs.readFileSync(`${filepath}/data/reviews.json`);
+    let reviews = JSON.parse(revData);
+    const reviewArray = [];
+
+    reviews.forEach(element => {
+        if(element.mealId == id) {
+            reviewArray.push(element);
+        }
+    });

```

Comment:
if you try out my suggestion you will see the same result and you can delete lines from 10 to 14 :)

Array.prototype functions are cool and easy to understand once you do them a couple of times. And, best of all, they are self explanatory. In this case I am using .filter function that will loop through every element of the reviews array and only return the ones that pass the if statement. If statement in this case is just checking if the review.id is equal to the id passed to the function.
If you by any chance get nothing in reviews array then make sure that both id passed to function and review.id are the same data type (integer). You should never use `==`!